src/app/core/filter/filter-generator/filter-generator.service.spec.ts

```diff
@@ -20,12 +19,14 @@ import { ConfigurableEnumFilter } from "../filters/configurableEnumFilter";
 import { EntityFilter } from "../filters/entityFilter";
 import { FormFieldConfig } from "../../common-components/entity-form/FormConfig";
 import { TestEntity } from "../../../utils/test-utils/TestEntity";
-import { EntitySchemaField } from "../../entity/schema/entity-schema-field";
+import {
+  EntitySchemaField,
+  PLACEHOLDERS,
+} from "../../entity/schema/entity-schema-field";
+import { CurrentUserSubject } from "app/core/session/current-user-subject";
 
 describe("FilterGeneratorService", () => {
-  let service: FilterGeneratorService;
-  let filterService: FilterService;
-
+  let service: FilterGeneratorService, filterService: FilterService;
```

Comment:
```suggestion
  let service: FilterGeneratorService;
  let filterService: FilterService;
  
```

Let's keep this a bit more verbose and clearly separated to make it easier to read and consistent across our code base.